package.json

```diff
@@ -134,6 +134,11 @@
             "type": "string"
           },
           "markdownDescription": "Optional list of custom resolvers passed to Coursier when fetching metals dependencies.\n\nFor documentation on accepted values see the [Coursier documentation](https://get-coursier.io/docs/other-repositories).\n\nThe extension will pass these to Coursier using the COURSIER_REPOSITORIES environment variable after joining the custom repositories with a pipe character (|)."
+        },
+        "metals.superMethodLenses": {
```

Comment:
Out of curiosity, will there be a situation where people specifically don't want this? I feel like it may be more common for someone to just disable code lenses rather than in individual feature that uses code lenses? I sort of feel like up to this point we don't off this level of customize-ability, but this starts down the path of more configuration. Do we want this?

Reply:
I'm all for not adding configuration, but code lenses are a weird beast and some of them can be more annoying than others.

This specific one would appear very often and cause a lot of shifting in the UI, which can be more annoying than the "Run"|"Debug" one (which is very rare)

Reply:
I think we should have this option, because it does heavily impact the user's experience. The same happened for VS Code java plugin, where there were immediately request for an option to disable them.  

I imagine some people might want to just use the shortcuts and it might just not be useful for them to have these code lenses. Besides, disabling all lenses would mean disabling them everywhere and they still might want to have some other like for example for git.

>but this starts down the path of more configuration

We do need some customization and this doesn't add any overhead really. I agree we need to be careful about it, but using a simple boolean value is nothing that will make us start with even more configuration.

Reply:
Sounds good 👍

Reply:
okay I will leave as it is then

Reply:
Shouldn't it be `superMethodLensesEnabled` ?

Reply:
correct, renamed :+1: